collector/internal/helpers.go

```diff
@@ -0,0 +1,236 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Can't we move `opentelemetry-ebpf-profiler/helpers.go` into its own package that we can import here to avoid duplicating all this code?

Reply:
Done in https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/197